biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/PomRepository.java

```diff
@@ -35,18 +35,21 @@
 	private final HttpClient		client;
 	private final PromiseFactory	promiseFactory;
 	final boolean					transitive;
+	final boolean					useDependencyManagement;
 
-	PomRepository(MavenRepository repo, HttpClient client, File location, boolean transitive) {
```

Comment:
Maybe you should not touch the existing constructor 	
`PomRepository(MavenRepository repo, HttpClient client, File location, boolean transitive)`

and delegate to the new added: 
`PomRepository(MavenRepository repo, HttpClient client, File location, boolean transitive,
		boolean useDependencyManagement)`

Reply:
But it is default access (package private), so it is it probably not worth it to have 2 constructors.

Reply:
I had the same thought as @stbischof but only tried to keep backward compatibility on API classes. I tend to site with BJ on this one.